<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpiderWindow</class>
 <widget class="QMainWindow" name="SpiderWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>937</width>
    <height>658</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>数据抓取</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>921</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>工具栏</string>
    </property>
    <widget class="QLabel" name="lblTitle">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>580</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>......</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnStart">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>30</y>
       <width>70</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>启动</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnStop">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>30</y>
       <width>70</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>停止</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnDownloadAll">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>760</x>
       <y>30</y>
       <width>70</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>下载所有</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnClose">
     <property name="geometry">
      <rect>
       <x>840</x>
       <y>30</y>
       <width>70</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>关闭</string>
     </property>
    </widget>
   </widget>
   <widget class="QTextEdit" name="txtLogs">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>921</width>
      <height>541</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
